Projekty/Projekt2/Deregowski_Jamrozy_Janus/P2_KM1_Deregowski_Jamrozy_Janus.ipynb

```diff
@@ -0,0 +1,3178 @@
+{
```

Comment:
<p>Lepiej byłoby taką analizę przeprowadzić w ramach KM3 w celu zweryfikowania jakości klasteringu, a teraz (w ramach KM1) założyć, że nie mamy informacji o klasach. Niemniej jednak nie było to jasno zasygnalizowane w poleceniu, więc takie podejście jest akceptowalne.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>